orm/errors.go

```diff
@@ -8,3 +8,7 @@ import (
 
 // ErrInvalidIndex is returned when an index specified is invalid
 var ErrInvalidIndex = errors.Register(100, "invalid index")
+
+// ErrBucketFixed is returned when already initialized bucket is tried
+// to be indexed again
+var ErrBucketFixed = errors.Register(101, "bucket fixed")
```

Comment:
Would it be better to call it `ErrBucket`, `ErrConflict` or `ErrIndex`? I think the _Fixed_ part does not belong here.